src/app/ui/AppProvider/ToastProvider.tsx

```diff
@@ -27,15 +39,70 @@ const Toast: FC<ToastProps> = ({ message }) => (
   </Box>
 );
 
-const ToastProvider: FC<PropsWithChildren> = ({ children }) => (
-  <RNTNToastProvider
-    offsetBottom={20}
-    renderToast={({ message }) => {
-      return <Toast message={message} />;
-    }}
-  >
-    {children}
-  </RNTNToastProvider>
-);
+const getDefaultBannerProps = (props: ToastProps) => {
+  let type = props.type?.replace('banner', '').toLowerCase(),
```

Comment:
I'm curious what is the purpose of this string replacement, can you explain?

I would just go with a default value of `success` if `type` is not supplied.